clang/test/CodeGenOpenCL/builtins-amdgcn.cl

```diff
@@ -839,6 +839,18 @@ unsigned test_wavefrontsize() {
   return __builtin_amdgcn_wavefrontsize();
 }
 
+// CHECK-LABEL test_get_fpenv(
```

Comment:
Ideally we would also warn if you used these without fenv access enabled

Reply:
Is this related to the DX10 clamp / traps potentially being disabled? Or is this an LLVM concept.

Reply:
It's a standard C concept. You need to enable #pragma STDC FENV_ACCESS ON to do anything valid with the floating point environment

Reply:
Hm, I'm not sure. I feel like this is just letting the user access the hardware directly which has a different use-case to the end user.

Reply:
This interface is 1 level removed from the hardware, but this cannot be legitimately used without fenv access enabled exactly the same way. It is not a different use case at all

Reply:
Added a sema check.